modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/app/App.js

```diff
@@ -358,15 +356,18 @@ class App extends EventEmitter {
 	 */
 	canNavigate(url) {
 		try {
-			const uri = new Uri(url);
+			const uri = url.startsWith('/')
+				? new URL(url, globals.window.location.origin)
```

Comment:
(not now) would be nice to get rid of these `globals` accesses.

Reply:
Yeah, I'll do a final run at `utils` and `globals` and things like that... the main problem is tests heavily depend on it, so it's a tough nut to crack 😢